r_practice/r_practice.md

```diff
@@ -0,0 +1,142 @@
+estimated_time: 1 hour
+
+@learning_objectives
+
+After completion of this module, learners will be able to:
+
+- Import a dataset from an online database
+- Recode data and change variable types in a dataframe
+- Use exploratory data visualization to identify trends in data and generate hypotheses
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# R Practice
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?:** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites:** Learners should be familiar with [the basics of R coding](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/r_basics_introduction/r_basics_introduction.md#1), including [data transformation with dplyr](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/r_basics_transform_data/r_basics_transform_data.md#1) and [data transformation with ggplot2](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/data_visualization_in_ggplot2/data_visualization_in_ggplot2.md#1). Learners should also have access to R, either on their own computer or in the cloud. 
+
+**Learning Objectives**:
+
+@learning_objectives
+
+</div>
+
+## Lesson Preparation
+
+For this module, you'll need access to R, either on your own computer or in the cloud. For details about how to download R, you can take a look at the [preparation section of our R Basics: Introduction module](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/r_basics_introduction/r_basics_introduction.md#5). If you'd prefer to work in the cloud, [posit.cloud](https://posit.cloud/) is a cloud-based R notebook environment. 
+
+## Topics to Review
+
+- Loading libraries 
+- Reading a CSV and creating a data frame
+- Editing/recoding data
+- Exploratory data visualization
+
+Stretch topics
+=====
+
+There are a couple of additional skills that haven't been introduced in a module yet, but that might come in handy solving this data science problem: 
+
+- Correlation matrices
+- Heat maps
+
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+Finding correlations and creating heat maps are not topics that we have taught you in the preceding modules-- which means you might need to do some searching to figure out how to do them! This is an important step in your development as a programmer and a data scientist, but don't worry if it seems difficult in the beginning; it'll get easier with practice. To start, try googling "R" and "correlations" and see where it takes you.
```

Comment:
```suggestion
Finding correlations and creating heat maps are not topics that you have learned in preceding modules or in other prior learning -- which means you might need to do some searching to figure out how to do them! This is an important step in your development as a programmer and a data scientist, but don't worry if it seems difficult in the beginning; it'll get easier with practice. To start, try googling "R" and "correlations" and see where it takes you.
```

Similar to Python, I'm decoupling this module from our previous modules in case this is ever offered separately (e.g. as a self-assessment at the beginning, who knows?)

Reply:
Oooh yeah makes sense!